docs/getting-started/integrations-cdn/angular.md

```diff
@@ -99,6 +99,73 @@ All available Angular versions are [listed on npm](https://www.npmjs.com/package
 
 This guide assumes you already have a Angular project. To create such a project, you can use Angular CLI. Refer to the [Angular documentation](https://angular.io/cli) to learn more.
 
+### Installing the Angular component from npm
+
+Install the [CKEditor&nbsp;5 WYSIWYG editor component for Angular](https://www.npmjs.com/package/@ckeditor/ckeditor5-angular):
+
+```bash
+npm install @ckeditor/ckeditor5-angular
+```
+
+### Using the Angular component
+
+To use CKEditor&nbsp;5 with CDN, you need to import the `loadCKEditorCloud` function and call it inside `ngOnInit` with the `version` provided in the configuration.
+
+```ts
+import { Component } from '@angular/core';
+import { loadCKEditorCloud, CKEditorCloudResult } from '@ckeditor/ckeditor5-integrations-common';
+import { ClassicEditor, EditorConfig } from 'https://cdn.ckeditor.com/typings/ckeditor5.d.ts';
+
+@Component( {
+	selector: 'app-root',
+	templateUrl: './app.component.html',
+	styleUrls: ['./app.component.css'],
+} )
+export class SimpleCdnUsageComponent {
+	public Editor: ClassicEditor | null = null;
+
+	public config: EditorConfig | null = null;
+
+	public editorData = '<p>Hello world!</p>';
+
+	public ngOnInit(): void {
+		loadCKEditorCloud( {
+			version: '43.0.0',
+			premium: true
+		} ).then( this._setupEditor.bind( this ) );
+	}
+
+	private _setupEditor ( cloud: CKEditorCloudResult ) {
+		const {
+			ClassicEditor,
+			Essentials,
+			Bold,
+			Italic,
+			Paragraph,
+			Undo,
+			Mention
+		} = cloud.CKEditor;
+
+		this.Editor = ClassicEditor;
```

Comment:
I fixed the snippet, and everything works except this line. Typescript shouts there is some type mismatch.
```
Type 'typeof ClassicEditor' is missing the following properties from type 'ClassicEditor': ui, destroy...
```

Reply:
There should be `public Editor: typeof ClassicEditor | null = null;` instead of `public Editor: ClassicEditor | null = null;`

Reply:
Maybe you'll debug it faster than me. @Mati365